src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
I believe it should be a factory.  The version we have in our cfmePOC actually has all the defaults for all the C3 chart types, so the file is injected into donut, sparkline, utilization, etc.. charts.
I will change it's filename.
-Thanks